### ISSUE TITLE
VimwikiBacklinks doesn't see wiki-style [[links]] when using markdown syntax

### ISSUE THREAD

**Issue by nebulaeandstars** [vimwiki/vimwiki#1244]
tl;dr: VimwikiBacklinks can recognise \[normal-links\](normal-links), but not [[wiki-style-links]]. Given that vimwiki seems to support both formats, I assume this is a bug.

- expected behaviour: both forms of link should be counted as a backlink.
- actual behaviour: only \[markdown-style links\](some-other-note) are detected.
- bug is present in the latest versions of both `dev` and `master`.

vimwiki config:
```vim
let g:vimwiki_list = [{
            \'path': '~/docs/notes/',
            \'path_html': '~/docs/notes/html',
            \'syntax': 'markdown',
            \'ext': '.md',
            \}]

let g:vimwiki_table_mappings = 0
let g:vimwiki_global_ext = 0
```

Steps to reproduce:
1. Make three files
2. Add two links to the first file:
    - `[second-file](second-file)`
    - `[[third-file]]`
3. Follow the links and test using `:VWB`
    - the second file will have a backlink to the first
    - the third file will not

**Comment by nebulaeandstars** [comment]
I'm not that familiar with vimscript, but I've made an MR for this, as it ended up being a fairly quick fix

**Comment by fcsm1922** [comment]
@nebulaeandstars,
> I'm not that familiar with vimscript, but I've made an MR for this, as it ended up being a fairly quick fix

Hi.Please tell me how you solved this problem?

**Comment by nebulaeandstars** [comment]
I just changed part of the function that generates the list of links.

Before it was saying "If in markdown mode, count markdown links, otherwise count wiki links."

Now it says "Count wiki links. If in markdown mode, also count markdown links."

Pull request is [here](https://github.com/vimwiki/vimwiki/pull/1245). It doesn't like it can merge anymore, though.

**Comment by fcsm1922** [comment]
@[nebulaeandstars](https://github.com/nebulaeandstars), 

did you change this in the base.vim file?
could you post your file? so as not to make mistakes when changing))

**Comment by fcsm1922** [comment]
made changes to the file ~/.vim/bundle/vimwiki/autoload/vimwiki/base.vim that you made.... and nothing..
and yes... the base.vim file is smaller for some reason (by the number of lines of code)

in the created vimwiki directory
[[folder/filename]]
when converting index.md to html file index.html outputs
[[folder/filename]] instead of a link to a file located in folder

VimwikiShowVersion: stable 2.5

**Comment by lumpsoid** [comment]
@fcsm1922 
you can use the fzf plugin for vim on this purpose, just take search by the name (id) of your file

**Comment by brennen** [comment]
Should be resolved with merge on 3ac8e1ae.
